### ISSUE TITLE
Center Loss optimizer confusion

### ISSUE THREAD

**Issue by okbalefthanded** [Want2Vanish/SincMSNet#2]
Hi,

Thanks for providing the paper's source code with all the details, I have a question regarding the Optimizer used for Center Loss. In the paper the Adam optimizer is mentioned as the one used for minimizing the loss function, yet in the code the optimzer4center object is created as an SGD optimizer and optimized separately. 
I've also checked the original code where the Center Loss was first introduced in the context of face recognition and the authors specified that the two methods are possible: one optimizer for both losses or a separate optimizers.
So which method has been used in your evaluation ?

Regards.

**Comment by Want2Vanish** [comment]
First of all, thank you for your interest in this work. Second, I apologize for not responding to messages in a timely manner.
In this work, we used different optimizers to optimize two different losses separately. Specifically, Adam is used to optimize the Cross entropy loss while the SGD optimizer is used to optimize the Center loss.
I did not attempt to use only one optimizer to optimize two different loss functions at the same time. The reason why I used SGD to optimize the center loss is that one of the code implementations of the Center loss that I referenced uses SGD.
